### PR TITLE
use strict

### PR DESCRIPTION
**Pull request by BridgeAR**
This is going to improve the speed and code safety a tiny bit by using [`use strict`] (https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Strict_mode) plus some style changes (adding semicolons). I added the later as I read that most of you seemed to prefer those. Using use strict in node should not break any code if it has not been broken before, since node encapsulates each file and does not apply it global.

### CODE REVIEW

File: benches/stress/codec.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-var MsgPack = require('node-msgpack');
-msgpack = {
-	encode: MsgPack.pack,
-	decode: function(str) { return MsgPack.unpack(new Buffer(str)); }
-};
+// var MsgPack = require('node-msgpack');
+// var msgpack = {
+// 	encode: MsgPack.pack,
+// 	decode: function(str) { return MsgPack.unpack(new Buffer(str)); }
+// };
```
Comment (erinspice) [inline benches/stress/codec.js]: Can you explain to someone not so familiar with the codebase why you commented this out? If it's not used at all, it should be deleted.

**Comment by erinspice** [comment]
I fully support this! I asked one question about some commented code.

File: benches/stress/codec.js (hunk 1 above)
Comment (BridgeAR) [inline benches/stress/codec.js]: `module.exports = msgpack;` is already commented out and therefor this is a unused variable. I was not sure what this part was for and normally I'd just delete unused code, but I'd prefer to know the reason for the original code before deleting it :).

Comment (erinspice) [inline benches/stress/codec.js]: Thanks!

**Comment by BridgeAR** [comment]
Rebased

**Comment by josephpage** [comment]
:+1:

**Pull request merged**